.travis.yml

```diff
@@ -1,61 +1,58 @@
-language: python
+language: generic
 sudo: false
-python:
-  - 2.7_with_system_site_packages
-  - 3.4
-  - 3.5
-env:
-  - ETS_TOOLKIT='qt4'
-  - ETS_TOOLKIT='wx'
-  - ETS_TOOLKIT='null'
-matrix:
-  exclude:
-    - python: 3.4
-      env: ETS_TOOLKIT='wx'
-    - python: 3.5
-      env: ETS_TOOLKIT='wx'
-    - python: 3.5
-      env: ETS_TOOLKIT='qt4'
-cache:
-  - pip
-  - ccache
+
 addons:
   apt:
     packages:
     - python-qt4
-    - python-qt4-gl
     - python-qt4-dev
+    - python-qt4-gl
     - python-wxtools
-    - python-numpy
-    - libjpeg8-dev
+    - ccache
+    - cmake
+    - swig
     - zlib1g-dev
     - libpng-dev
     - libfreetype6-dev
-    - python-cairo
-    - cmake
-    - swig
-    - ccache
+    - libcairo2-dev
+
+env:
+  global:
+    - INSTALL_EDM_VERSION=1.5.2
+      PYTHONUNBUFFERED="1"
+
+matrix:
+  include:
+  - env: RUNTIME=2.7 TOOLKIT=wx
+  - env: RUNTIME=2.7 TOOLKIT=pyqt
```

Comment:
No particular one, other than that I looked at the current Travis file and misread what it does. I'll add PySide.